docs/operating-scylla/admin-tools/sstabledump.rst

```diff
@@ -1,8 +1,10 @@
 SSTabledump
 ============
 
-This tool allows you to converts SSTable into a JSON format file.
-SSTabledump supported when using Scylla 3.0, Scylla Enterprise 2019.1, and newer versions.
+SSTabledump allows you to converts SSTable into a JSON format file.
+This tool is supported when using Scylla 3.0, Scylla Enterprise 2019.1, and newer versions.
+It is deprecated since Scylla 5.3, and will be removed in a future release.
+Please consider switching to :doc:`Scylla SSTable </operating-scylla/admin-tools/scylla-sstable>`.
```

Comment:
I think you should use a `..note` or maybe even a `..warning` to make this stand out.
See example code here: https://github.com/scylladb/scylladb/blob/master/docs/operating-scylla/admin-tools/select-from-mutation-fragments.rst?plain=1
Rendered here: https://opensource.docs.scylladb.com/master/operating-scylla/admin-tools/select-from-mutation-fragments.html

Reply:
sure.